src/Discord.Net.Rest/API/Rest/CreateGuildChannelParams.cs

```diff
@@ -10,9 +10,20 @@ internal class CreateGuildChannelParams
         public string Name { get; }
         [JsonProperty("type")]
         public ChannelType Type { get; }
```

Comment:
This field's actually optional. 
<https://discordapp.com/developers/docs/resources/guild#create-guild-channel>
>All parameters for this endpoint are optional excluding `name`